suitcase/hdf5.py

```diff
@@ -2,21 +2,32 @@
 import warnings
 import h5py
 import json
+import copy
 from metadatastore.commands import get_events_generator
 from databroker.databroker import fill_event
 from databroker.core import Header
 
 
 __version__ = "0.2.2"
 
-def export(headers, filename):
+def export(headers, filename, stream_name=None, fields_unwanted=None, timestamps_opt=True):
     """
+    Create hdf5 file to preserve the structure of databroker. Necessary copy are made at each step, so
+    headers will not be modified.
+
     Parameters
     ----------
     headers : a Header or a list of Headers
         objects retruned by the Data Broker
     filename : string
         path to a new or existing HDF5 file
+    stream_name : string, optional
+        None means save all the data from each descriptor, i.e., user can define stream_name as primary,
+        so only data with descriptor.name == primary will be saved.
+    fields_unwanted : list, optional
```

Comment:
I am aware of the fields kws. But it may be easy to do the inverse way here. The use case is like I want to include data from databroker as many as possible, but only want to exclude one or two fields, such as CCD image, if I mainly pay attention to fluorescence data.